daemon/response_test.go

```diff
@@ -107,3 +108,33 @@ func (s *responseSuite) TestRespJSONWithNullResult(c *check.C) {
 	c.Assert(err, check.IsNil)
 	c.Check(string(data), check.Equals, `{"type":"","status-code":0,"status":"","result":null}`)
 }
+
+type xT struct {
+	called int
```

Comment:
What's the difference between called and count?

Reply:
called is how many time the function was called, count is one of the two values the funciton returns